cmd/influx/main.go

```diff
@@ -249,6 +258,22 @@ func (c *CommandLine) SetFormat(cmd string) {
 	}
 }
 
+func (c *CommandLine) dump() {
+	response, err := c.Client.Dump(c.Database)
+	defer response.Close()
+	if err != nil {
+		fmt.Printf("Dump failed. %s\n", err)
+	} else {
+		scanner := bufio.NewScanner(response)
```

Comment:
The response is line-oriented, so this felt like a natural way to do it.
The code works. 

I will change it if you insist.